project/src/components/header/header.tsx

```diff
@@ -1,20 +1,19 @@
 import Logo from '../logo/logo';
+import UserBlock from '../user-block/user-block';
 
-function Header() : JSX.Element {
+type HeaderProps = {
+  heading?: string;
+  isUserPage?: boolean;
+  isUserBlock?: boolean
+  isFilmCard?: boolean;
+}
+
+function Header({heading, isUserPage, isUserBlock, isFilmCard }: HeaderProps) : JSX.Element {
```

Comment:
Смотри за форматированием, пробелы около фигурных скобок либо ставим либо нет
```suggestion
function Header({ heading, isUserPage, isUserBlock, isFilmCard }: HeaderProps) : JSX.Element {
```